spec/requests/cats_spec.rb

```diff
@@ -2,6 +2,107 @@
+        image: 'https://hips.hearstapps.com/hmg-prod/images/cute-cat-photos-1593441022.jpg'
+      )
+
+      get '/cats'
+
+      cat = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      expect(cat.length).to eq 1
+    end
+  end
+
+  describe "POST /create" do
+    it "creates a cat" do
+      cat_params = {
+        cat: {
+          name: 'Buster',
+          age: 4,
+          enjoys: 'Meow Mix, and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+
+      post '/cats', params: cat_params
+
+      expect(response).to have_http_status(200)
+
+      cat = Cat.first
+
+      expect(cat.name).to eq 'Buster'
+      expect(cat.age).to eq 4
+      expect(cat.enjoys).to eq 'Meow Mix, and plenty of sunshine.'
+      expect(cat.image).to eq 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+    end
+  end
+
+  describe "PATCH /update" do
+    it "updates a cat" do
+      cat_params = {
+        cat: {
+          name: 'Buster',
+          age: 4,
+          enjoys: 'Meow mix, and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+
+      post '/cats', params: cat_params
+      cat = Cat.first
+      
+      updated_params = {
+        cat: {
+          name: 'Felix',
+          age: 4,
+          enjoys: 'Meow mix, and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+      
+      patch "/cats/#{cat.id}", params: updated_params
+
+      updated_cat = Cat.find(cat.id)
+      expect(response).to have_http_status(200)
+      expect(updated_cat.name).to eq "Felix"
+
+    end
+  end
+
+  describe "DESTROY /delete" do
+    it "deletes a cat" do
+      cat_params = {
+        cat: {
+          name: 'Buster',
+          age: 4,
+          enjoys: 'Meow mix, and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+
+      post '/cats', params: cat_params
+      cat = Cat.first
+
+      deleted_params = {
+        cat: {
+          name: 'Buster',
+          age: 4,
+          enjoys: 'Meow mix, and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
```

Comment:
Because this code is not necessary to perform the delete functionality, please remove.